reference/HA.md

```diff
@@ -0,0 +1,12 @@
+## High Availability 
+
+Mayastor 2.0 brings in the High Availability (HA) feature. In the event of a failure, this feature will guarantee I/O continuity. 
```

Comment:
Is "guarantee" too strong of a statement @avishnu ?

Reply:
Yes, seems so. Something on the lines "this feature attempts to ensure I/O continuity" ?